.github/workflows/cron.yml

```diff
@@ -96,6 +96,10 @@ jobs:
             ${{ runner.os }}-
       - name: Install tox
         run: pip install tox
+      - name: Uninstall pip
+        if: ${{ matrix.os }} == Windows
+        run: python -m pip uninstall pip
```

Comment:
Could you _temporarily_ add the `main` branch to `ci.yaml` matrix, to make sure the fix works?